tests/integration/mounts_sshfs.bats

```diff
@@ -31,16 +31,16 @@ function setup_sshfs() {
 		-o StrictHostKeyChecking=no
 		-o PasswordAuthentication=no"
 
-	if ! [ -v DIR ]; then
-		DIR="$BATS_RUN_TMPDIR/fuse-sshfs"
-		mkdir -p "$DIR"
-		# Make sure we clear all superblock flags to make sure bind-mounts can
-		# unset these flags.
-		if ! $sshfs -o rw,suid,dev,exec,atime rootless@localhost: "$DIR"; then
-			# fallback to tmpfs if running in without sshfs
-			mount -t tmpfs -o rw,suid,dev,exec,diratime,strictatime tmpfs "$DIR"
-		fi
+	DIR="$BATS_RUN_TMPDIR/fuse-sshfs"
+	umount "$DIR" 2>/dev/null || true
+	mkdir -p "$DIR"
+	# Make sure we clear all superblock flags to make sure bind-mounts can
+	# unset these flags.
+	if ! $sshfs -o rw,suid,dev,exec,atime rootless@localhost: "$DIR"; then
+		# fallback to tmpfs if running in without sshfs
+		mount -t tmpfs -o rw,suid,dev,exec,diratime,strictatime tmpfs "$DIR"
 	fi
+
 	# Reset atime flags. "diratime" is quite a strange flag, so we need to make
 	# sure it's cleared before we apply the requested flags.
 	mount --bind -o remount,diratime,strictatime "$DIR"
```

Comment:
We probably don't need this (all works fine without this), but I didn't change it here to show the minimal diff I needed.

Reply:
If we switch to doing `umount` this is not longer needed. The reason for this line was (as explained in the comment) that we need to explicitly force the atime to be `relatime` using this incantation when doing remounts.

Reply:
Removed this line